core/templates/dev/head/pages/splash/splash_nv1.html

```diff
@@ -0,0 +1,260 @@
+{% extends 'pages/base.html' %}
+
+{% block prerender %}
+  <link rel="prerender" href="/library">
+{% endblock prerender %}
+
+{% block maintitle %}
+  I18N_SPLASH_PAGE_TITLE
+{% endblock maintitle %}
+
+{% block header_js %}
+  {{ super() }}
+{% endblock %}
+
+{% block navbar_breadcrumb %}
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div ng-controller="Splash" style="background-color: #afd2eb; font-size: 14px;">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-4 oppia-display-desktop-only col-sm-3">
+          <img ng-src="<[aboutPageMascotImgUrl]>" id="oppia-splash-new-otter" />
+        </div>
+        <div class="col-xs-12 col-sm-9 col-md-6" style="margin-top: 20px;">
+          <h1 class="oppia-splash-new-h1">Welcome to Oppia</h1>
+          <h2 class="oppia-splash-new-h2">Oppia's mission is to help anyone learn in an effective and enjoyable way.</h2>
+          <div class="oppia-button-container">
+            <button type="button"
+                    ng-click="onClickBrowseLibraryButton()"
+                    class="btn oppia-transition-200 oppia-splash-new-cta-button">
+                    Start Learning
+            </button>
+            <button type="button"
+                    ng-click="onRedirectToLogin('/dashboard?mode=create')"
+                    class="btn oppia-transition-200 oppia-splash-new-cta-button">
+                    Share Your Knowledge
```

Comment:
Decrease indentation. It should be two spaces from the opening of the button tag. I think this is generally the convention followed in oppia.